assets/sass/04-elements/links.scss

```diff
@@ -0,0 +1,24 @@
+/*
+ * text-underline-offset doesn't work in Chrome at all 👎
+ * But looks nice in Safari/Firefox, so let's keep it and
+ * maybe Chrome will support it soon.
+ */
+a {
+	color: var(--global--color-black);
+	text-underline-offset: 3px;
+	transition: all .15s ease-in-out;
+  }
+
+a:hover {
+	text-decoration-style: dotted;
+}
+
+/*
+ * The reason for the !important style here is the global focus styles created
+ * by the `.site *:focus {` rules in the 03-generic/reset.scss file.
+ */
+a:focus {
+	outline: none !important;
```

Comment:
We can not use outline none for accessibility reasons, to work on high contrast mode, you can set this to transparent instead.